pages/login.vue

```diff
@@ -56,22 +65,39 @@
 
 <script>
 
+import {
+  BFormInput, BFormGroup, BForm, BButton,
+} from 'bootstrap-vue';
+
 export default {
-  name   : 'Login',
-  layout : 'login',
-  methods: { goToSCBDLogin, goToSCBDSignUp },
+  name      : 'Login',
+  layout    : 'login',
+  auth      : 'guest',
+  components: {
+    BFormInput,
+    BFormGroup,
+    BForm,
+    BButton,
+  },
+  data() {
+    return {
+      userInfo: {
+        email   : null,
+        password: null,
+      },
+    };
+  },
+  methods: {
+    login,
+  },
 };
 
-function goToSCBDLogin() {
-  const returnUrl = window.location.href;
-
-  window.location.href = `${this.$ssoScbd.auth.accountsUrl}?returnUrl=${encodeURIComponent(returnUrl.replace('/login', '/'))}`;
+async function login() {
+  try {
+    await this.$auth.loginWith('local', { data: this.userInfo });
+    this.$toast.success('Logged In!');
+  } catch (err) {
+    this.$toast.error('Incorrect Credentials!');
```

Comment:
Can you also show/add a `warning` alert box into the form?  Below *Forget Password* [Login] button

---------------
Email
[ ]
Password
[ ]
Forget Password?    [Login]
>  **Login failed** 
> The email address and password combination you entered is incorrect.
---------------

clear it when login() is trigger OR after 5 seconds, using the automatic CAlert timer of `show = 5`.

https://coreui.io/vue/docs/components/alert.html


Alose change toast text
```suggestion
    this.$toast.error('The email address and password combination you entered is incorrect');
```

Reply:
DONE!